dynawo/3rdParty/libxml/sax/parser/include/xml/sax/parser/ParserFactory.h

```diff
@@ -53,6 +53,8 @@ class ParserFactory {
    * @brief allocates a new parser.
    */
   Parser* newParser() const;
+private:
+  bool multiThreadingMode_;
```

Comment:
Proposition: use a default value to false for this member. This leads to less modifications

Reply:
I did this on purpose to force the developer to think about whether this is called in multi-threading or not and to explicitely add the init and terminate of xerces in his application if  required